datafusion/core/tests/sql/explain_analyze.rs

```diff
@@ -792,7 +811,6 @@ async fn csv_explain() {
               \n        CsvExec: files=[ARROW_TEST_DATA/csv/aggregate_test_100.csv], has_header=true, limit=None, projection=[c1, c2]\
               \n"
         ]];
-    assert_eq!(expected, actual);
 
     // Also, expect same result with lowercase explain
```

Comment:
This comment seems to be out of date -- the different is not just just `explain` vs `EXPLAIN` it is also the explicit cast that was added